test/Main.purs

```diff
@@ -0,0 +1,11 @@
+module Test.Main where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Class.Console (log)
+
+main :: Effect Unit
+main = do
+  log "🍝"
+  log "You should add some tests."
```

Comment:
I'm not sure what to do about the tests. Ideally we could run the editor, manipulate it, and verify the result is what we expect. But that might require something more involved, like a Puppeteer test, which I'm not particularly interested in maintaining.